app/src/chat_engine.py

```diff
@@ -395,21 +407,19 @@ class ImagineLaEngine(BaseEngine):
 - Respond in the same language as the user's message.
 - If the user asks for a list of programs or requirements, list them all, don't abbreviate the list. \
 For example "List housing programs available to youth" or "What are the requirements for students to qualify for CalFresh?"
-- If your answer involves recommending going to a DPSS location, provide this link in your answer: https://dpss.lacounty.gov/en/resources/offices.html
-- If your answer involves recommending contacting DPSS, provide this link in your answer: https://dpss.lacounty.gov/en/resources/contact.html
 
 Provide citation numbers:
 - When referencing the context, do not quote directly. Use the provided citation numbers (e.g., (citation-1)) to indicate when \
 you are drawing from the context. To cite multiple sources at once, you can append citations like so: (citation-1) (citation-2), etc. \
 For example: 'This is a sentence that draws on information from the context. (citation-1)'
 
 Example question:
-Can a client get Unemployment and disability at the same time?
+Can my client get Unemployment and disability at the same time?
 
 Example Answer:
-No, a client cannot receive Unemployment Insurance (UI) and State Disability Insurance (SDI) benefits at the same time. (citation-1)
-They must choose the program that best fits their situation. If they don't know which program to apply for, \
-they can apply for both, and their eligibility for each will be reviewed. (citation-2) (citation-3)"""
+No, your client can’t get Unemployment Insurance (UI) and State Disability Insurance (SDI) at the same time. (citation-1)
+They need to choose the one that works best for their situation. If they’re not sure which one to apply for, \
+they can apply for both, and the state will check if they qualify for either one. (citation-2) (citation-3)"""
```

Comment:
Sage page moved the period _after_ the `(citation-N)` text. I moved it before for consistency since it was decided that citation links should occur after the punctation.